build.gradle.kts

```diff
@@ -10,7 +10,7 @@ plugins {
 
 allprojects {
   group = "uk.gov.justice.service.hmpps"
-  version = "2.0.1"
+  version = "2.0.2-beta"
```

Comment:
wasn't sure whether to to use `beta` here or not in the new version

Reply:
I think it's best - gone up a version of minor version of both Kotlin and Spring, can't hurt to be cautious.